### PR TITLE
Removing the mention of multiple authors

### PR DESCRIPTION
**Pull request by fe-lix-**
## Description

Simplifying the configuration example as it creates confusion.

## Motivation and Context

I received feedback from several users, the previous version of the configuration did not help them.

### CODE REVIEW

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 1]
```diff
@@ -131,16 +131,9 @@ This is sample asset picker configuration file that allows filtering assets by i
 
 ```
 {
-    "aemTierType": [
-        "delivery",
-        "author"
-    ],
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:137]: The default configuration is the correct and should also not be modified by most users.

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 2]
```diff
@@ -131,16 +131,9 @@ This is sample asset picker configuration file that allows filtering assets by i
...
-    "apiKey": "asset_search_service",
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:138]: This is irrelevant to our users.

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 3]
```diff
@@ -131,16 +131,9 @@ This is sample asset picker configuration file that allows filtering assets by i
...
     "repoNames": [
       "author-p7452-e733061.adobeaemcloud.com",
-      "author-p7452-e12437.adobeaemcloud.com",
       "delivery-p130360-e1272151.adobeaemcloud.com",
-      "author-p130360-e1272151.adobeaemcloud.com"
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:143]: You can only pick from a single author instance, removing this duplicity.

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 4]
```diff
@@ -15,10 +15,7 @@ It's useful in case where we want to enable authors to select assets of certain
 
 This extension enhances [Universal Editor](../../../services/aem-universal-editor/) properties panel for media elements. It replaces standard asset picker with a custom version of asset picker, with provided configuration applied while selecting assets.
 
-Following type of configuration is available:
-- AEM tier type ("delivery", "author" etc.)
-- Repo names to select assets from
-- ApiKey
+The extension will enable you to list the repositories your authors will be able to pick from.
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:18]: Rephrasing to account for the changes in the configuration example.

**Review by AmyJZhao — APPROVED** [review]

**Review by kokoc — APPROVED** [review]

**Review by nitinja — APPROVED** [review]

**Pull request merged**